api/util/logging.py

```diff
@@ -1,27 +1,73 @@
 import json
 import logging
+import sys
+from datetime import datetime
+from typing import Any
+
+from django.conf import settings
+from gunicorn.config import Config
+from gunicorn.glogging import Logger as GunicornLogger
 
 
 class JsonFormatter(logging.Formatter):
     """Custom formatter for json logs."""
 
-    def format(self, record):
+    def get_json_record(self, record: logging.LogRecord) -> dict[str, Any]:
+        formatted_message = record.getMessage()
+        return {
+            "levelname": record.levelname,
+            "message": formatted_message,
+            "timestamp": self.formatTime(record, self.datefmt),
+            "logger_name": record.name,
+            "process_id": record.process,
+            "thread_name": record.threadName,
+        }
+
+    def format(self, record: logging.LogRecord) -> str:
```

Comment:
The docstring of this function is now out of date.

Reply:
Thanks, removed!